posthog/api/action.py

```diff
@@ -355,22 +361,33 @@ def calculate_trends(filter: Filter, team_id: int, actions: QuerySet) -> List[Di
     return entities_list
 
 
-def calculate_retention(filter: Filter, team: Team, start_entity: Optional[Entity] = None, total_days=11):
+def calculate_retention(filter: Filter, team: Team, period: str, start_entity: Optional[Entity] = None, total_time=11):
```

Comment:
This function signature is clunky, eg having to pass a `period` string and a `unit` string of `total_time`.
Is there a reason for simply not taking a `timedelta` object directly?